frontend/src/components/match/stats/EditMinStats.js

```diff
@@ -62,32 +62,31 @@ const EditStats = () => {
     return team?.image ?? team?.image_url;
   };
 
+  const invalidateQueries = () => {
+    queryClient.invalidateQueries({ queryKey: ["match", params.matchId] });
+    queryClient.invalidateQueries({
+      queryKey: ["match-stats", params.matchId]
+    });
+  };
+
   const matchStatsHalfTimeMutation = createMutation({
     mutationFn: matchStatsHalfTime,
-    onSuccess: () => {
-      queryClient.invalidateQueries({ queryKey: ["match", params.matchId] });
-    }
+    onSuccess: () => invalidateQueries()
```

Comment:
nit: This can just be `onSuccess: invalidateQueries` right ? 
1 more small change :sweat_smile:

Reply:
hehe saw it comingg

Reply:
pushed